sdk/appcenter-auth/src/main/java/com/microsoft/appcenter/auth/Auth.java

```diff
@@ -306,7 +306,7 @@ private synchronized void setInstanceConfigUrl(String configUrl) {
     private synchronized void removeTokenAndAccount() {
         AuthTokenContext authTokenContext = AuthTokenContext.getInstance();
         removeAccount(authTokenContext.getHomeAccountId());
-        authTokenContext.setAuthToken(null, null, null);
```

Comment:
We don't need to modify the auth context since there is no getUser.
The only time we need access token is when the public API is called.
Many changes have thus to be reverted here.